backend/src/main/java/com/quashbugs/quash/controller/UserController.java

```diff
@@ -74,16 +69,14 @@ public ResponseEntity<ResponseDTO> getUserVerified(Authentication authentication
      */
     @PatchMapping
     public ResponseEntity<ResponseDTO> updateUser(Authentication authentication, @RequestBody User updatedUser) {
-        if (authentication != null && authentication.isAuthenticated()) {
+        try {
             User user = (User) authentication.getPrincipal();
             userService.updateUser(user, updatedUser);
```

Comment:
What happens in this case when the "user" object is null? when the "user" object is null, will the client know that no action has been taken?